.github/workflows/publish-ci.yml

```diff
@@ -98,3 +106,9 @@ jobs:
           user: __token__
           password: ${{ secrets.PYDEEPHAVEN_PYPI_TOKEN }}
           packages_dir: py/client/build/wheel/
+
+      - name: Publish @deephaven/jsapi-types to npmjs
+        if: ${{ startsWith(github.ref, 'refs/heads/release/v') }}
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.DEEPHAVENBOT_NPM_TOKEN }}
+        run: npm publish --tag latest web/client-api/types/build/deephaven-jsapi-types-*.tgz
```

Comment:
Does it make sense to add this to our GH release asset? If so, can you please update RELEASE.md (I think there will be a couple places) and make sure to note the updated procedure to `#dhc-release` channel?

Reply:
I think I've updated those sections, see if I missed anything?